pages/issue/issueCompany.tsx

```diff
@@ -72,28 +71,42 @@ export default function IssueCompany(props: any) {
       })
       .catch(function (error) {
         console.log(error);
+      })
+      .finally(() => {
+        setIsProcessing(false);
       });
   };
 
   const handleRejectCompanyIssuance = async (
     application: CompanyApplication,
   ) => {
+    setIsProcessing(true); // Set processing state to true
+
     try {
-      // Update database with credential issuance
-      await updateApplicationStatusInDb(
-        COLLECTIONS.COMPANY_APPLICATIONS,
-        application.address + "-" + application.timestamp,
-        APPLICATION_STATUS.REJECTED,
-      );
-      const updatedApplications: any = await getApplicationsFromDb(
-        COLLECTIONS.COMPANY_APPLICATIONS,
-      );
-      setApplications(updatedApplications);
+      // Call the API to update the application status in the database
```

Comment:
`updateApplicationStatusInDb` was failing because this call is from front-end and the mong-db is on the server side. Hence , needed an API endpoint. 
It was working before because firebase server can be accessed both form frontend as well as server.